api/cloud.proxy.js

```diff
@@ -235,6 +240,45 @@ IoTKitCloud.prototype.catalog = function (callback) {
     });
 };
 
+IoTKitCloud.prototype.pullActuations = function () {
+    var me = this;
+    var data = {
+        deviceToken: me.secret.deviceToken,
+        deviceId: me.deviceId,
+        accountId: me.secret.accountId
+    };
+    configurator.getLastActuationsPull(function(from) {
+        if(from) {
+            me.logger.info("Pulling only actuations from last pull time: " + new Date(from * 1000));
+            data.from = from;
+        }
+        me.proxy.pullActuations(data, function (result) {
+            if (result) {
+                me.logger.info("Running actuations");
+                var udp = updServer.singleton(conf.listeners.udp_port, me.logger);
+                var receiverInfo = {
+                    port: conf.receivers.udp_port,
+                    address: conf.receivers.udp_address
+                };
+                for (var i = 0; i < result.length; i++ ){
+                    var actuation = result[i];
+                    me.logger.info('Received actuation content: ' + JSON.stringify(actuation));
+                    var comp = me.store.byCid(actuation.componentId);
+                    var udpMessage = {
+                        component: comp.name,
+                        command: actuation.command,
+                        argv: actuation.params
+                    };
+                    me.logger.debug("Sending actuation: " + JSON.stringify(udpMessage));
```

Comment:
change this logger line to info instead of debug. NOTE: this change might require changes in e2e actuations tests,please check and if so create JIRA task to update them